src/pages/komodo-defi-framework/api/common_structures/activation/index.mdx

```diff
@@ -214,11 +214,11 @@ The `TendermintNode` object includes the following items for a given coin or tok
 
 The `SwapV2Contracts` object includes the following items for a given coin or token:
 
-| Parameter                      | Type   | Description                                                                                                                       |
-| ------------------------------ | ------ | --------------------------------------------------------------------------------------------------------------------------------- |
-| maker\_swap\_v2\_contract      | string | Address for the maker's new V2 swap smart contract. Must be provided if "use\_trading\_proto\_v2"is true in mm2 configuration     |
-| taker\_swap\_v2\_contract      | string | Address for the taker's new V2 swap smart contract. Must be provided if "use\_trading\_proto\_v2"is true in mm2 configuration     |
-| nft\_maker\_swap\_v2\_contract | string | Address for the maker's new V2 NFT swap smart contract. Must be provided if "use\_trading\_proto\_v2"is true in mm2 configuration |
+| Parameter                      | Type   | Description                                                                                                                                                                                          |
+| ------------------------------ | ------ | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| maker\_swap\_v2\_contract      | string | Address for the maker's new V2 swap smart contract. Must be provided if "use\_trading\_proto\_v2"is true in your [MM2.json file](/komodo-defi-framework/setup/configure-mm2-json/) configuration     |
```

Comment:
wasn't there a transition from mm.json to kdf.json ? if so, we should update this URL and links to it etc

Reply:
Not yet
```
smk@jarrah:~/mm2_777$ cp MM2.json KDF.json
smk@jarrah:~/mm2_777$ nano KDF.json
smk@jarrah:~/mm2_777$ mv MM2.json MM2_jag.json
smk@jarrah:~/mm2_777$ ./kdf
30 03:15:30, mm2:294] Komodo DeFi Framework 2.3.0-beta DT 
30 03:15:30, mm2:297] File '/home/smk/.kdf/MM2.json' is not present.
```